src/scripts/modules/window-with-sidebar/window-with-sidebar.less

```diff
@@ -50,7 +50,11 @@
     > .main {
       width: ~"calc(100% -" @sidebar-width + @fudge ~")";
       @media (max-width: 640px) {
-        visibility:hidden;
+        position: absolute; 
```

Comment:
how about using `display: none` instead of `visibility: hidden`? That would remove it from the flow of the document.

Reply:
Done. I've had to change `display: inline-block;` to `block` for `.main` and now it's not "jumping".